src/main/java/net/coderbot/iris/mixin/MixinShaderInstance.java

```diff
@@ -74,4 +76,13 @@
 
 		logger.warn(message, arg1, arg2);
 	}
+
+	@Redirect(method = "<init>", at = @At(value = "INVOKE", target = "Lcom/mojang/blaze3d/shaders/Uniform;glBindAttribLocation(IILjava/lang/CharSequence;)V"))
+	public void iris$redirectBindAttributeLocation(int i, int j, CharSequence charSequence) {
+		if (((Object) this) instanceof ExtendedShader && Arrays.asList("Position", "Color", "Normal", "UV0", "UV1", "UV2").contains(charSequence)) {
```

Comment:
List#contains is inefficient, not to mention constructing a new time every time we call bind. Best to create a HashSet / ImmutableSet once & call contains using that instead.

Reply:
Done.